packages/main/src/backend/import/__tests__/proxy-integration.test.ts

```
@@ -0,0 +1,32 @@
+import { afterAll, beforeEach, describe, expect, test, vi } from 'vitest';
+
+describe('Proxy support integration test', () => {
+  const originalEnv = process.env;
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+    // Reset environment
+    process.env = { ...originalEnv };
+    delete process.env.HTTPS_PROXY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)